source/http_filesystem/http_filesystem.go

```diff
@@ -0,0 +1,128 @@
+package httpfs
```

Comment:
rename the folder and file to be the same as the package name. e.g. `http_filesystem` -> `httpfs`

Reply:
Still need to rename the file